server/app/views/admin/programs/ManageProgramAdminsView.java

```diff
@@ -64,72 +71,105 @@ public Content render(
             .setTitle(fullTitle)
             .addMainContent(
                 renderHeader(fullTitle),
-                adminEmailTemplate(),
-                renderAdminForm(request, program.id(), existingAdminEmails));
+                renderHeader("Add New Admin"),
+                renderAddNewButton(request, program.id()),
+                renderHeader("Existing Admins"),
+                renderExistingAdmins(request, program.id(), existingAdminEmails),
+                renderBackButton());
 
     message.map(m -> m.setDuration(6000)).ifPresent(htmlBundle::addToastMessages);
 
     return layout.renderCentered(htmlBundle);
   }
 
-  /**
-   * Render a form with inputs for program admin emails. If program admins exist, the input fields
-   * will be pre-populated with their email addresses.
-   */
-  private FormTag renderAdminForm(
+  private DivTag renderAddNewButton(Http.Request request, long programId) {
+    FormTag formTag =
+        form()
+            .withMethod("POST")
+            .withAction(routes.ProgramAdminManagementController.add(programId).url());
+    FieldWithLabel emailField =
+        FieldWithLabel.email()
+            .setId("admin-email-input")
+            .setFieldName("adminEmail")
+            .setLabelText("Admin email address");
+    return div()
+        .with(
+            formTag.with(
+                emailField.getInputTag(),
+                makeCsrfTokenInputTag(request),
+                submitButton("Add")
+                    .withId("add-admin-button")
+                    .withClasses(ButtonStyles.SOLID_BLUE, "ml-2", "mb-6")))
+        .withClasses("border", "border-gray-300", "shadow-md", "w-1/2", "mt-6");
+  }
+
+  private DivTag renderExistingAdmins(
       Http.Request request, long programId, ImmutableList<String> existingAdminEmails) {
-    DivTag emailFields =
-        div()
-            .withId(EMAIL_CONTAINER_DIV_ID)
-            .withClasses("ml-4")
-            .with(each(existingAdminEmails, email -> adminEmailInput(Optional.of(email))))
+    return div(
```

Comment:
optional: I think you can do `div().with(table() ....);` instead of passing the table to the div and that might be more readable.

Reply:
Done